requirements.txt

```diff
@@ -1 +1,2 @@
 aiohttp
+attrs
```

Comment:
Let's not add a dependency for this. Use a namedtuple instead.

Reply:
Alright I just switched to namedtuple instead of attrs